python/tests/test_highlevel.py

```diff
@@ -1407,6 +1407,9 @@ def test_deprecated_apis(self):
         self.assertEqual(
             ts.get_pairwise_diversity(samples), ts.pairwise_diversity(samples))
         self.assertTrue(np.array_equal(ts.get_samples(), ts.samples()))
+        # Test APIs for a single tree in a ts
+        self.assertEqual(ts.first().get_length, ts.first().span)
+        self.assertEqual(ts.first().length, ts.first().span)
```

Comment:
Be nice to check this somewhere on on not just the first tree. Also, check 
```
l, r = tree.interval
tree.span == r - 
```

Reply:
Yep - I've just re-written it (and put it in a more sensible place). Thanks.